src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -108,14 +108,18 @@ SMALL_RECT Terminal::_GetSelectionRow(const SHORT row, const COORD higherCoord,
 }
 
 // Method Description:
-// - Get the current anchor position
+// - Get the current anchor position relative to the whole text buffer
 // Arguments:
 // - None
 // Return Value:
 // - None
 const COORD Terminal::GetSelectionAnchor() const
 {
-    return _selectionAnchor;
+    COORD selectionAnchorPos;
+    selectionAnchorPos.X = _selectionAnchor.X;
+    selectionAnchorPos.Y = _selectionAnchor.Y + gsl::narrow<short>(ViewStartIndex());
+
+    return selectionAnchorPos;

```

Comment:
@carlos-zamora please review selection changes

Reply:
Reviewed. We don't use this for our own selection. This is more for UiaData.